xml/System.IO.Pipelines/PipeWriter.xml

```diff
@@ -160,10 +160,10 @@
         <Parameter Name="cancellationToken" Type="System.Threading.CancellationToken" Index="1" FrameworkAlternate="dotnet-plat-ext-3.0" />
       </Parameters>
       <Docs>
-        <param name="source">To be added.</param>
-        <param name="cancellationToken">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="source">The stream from which the contents will be copied.</param>
+        <param name="cancellationToken">The token to monitor for cancellation requests. The default value is <see cref="P:System.Threading.CancellationToken.None" />.</param>
+        <summary>Asynchronously reads the bytes from the specified stream and writes them to the <see cref="T:System.IO.Pipelines.PipeWriter" />.</summary>
+        <returns>A task that represents the asynchronous copy operation.</returns>
         <remarks>To be added.</remarks>
       </Docs>
     </Member>
```

Comment:
Can you replace lines 219-222 with the following:
```
<param name="sizeHint">The minimum length of the returned <see cref="T:System.Memory`1" />. If 0, a non-empty memory buffer is returned.</param>
<summary>Returns a <see cref="T:System.Memory`1" /> to write to that is at least the requested size, as specified by the <paramref name="sizeHint" /> parameter.</summary>
<returns>A memory buffer of at least <paramref name="sizeHint" /> bytes. If <paramref name="sizeHint" /> is 0, returns a non-empty buffer.</returns>
<remarks>  
     <format type="text/markdown"><![CDATA[  

There is no guarantee that successive calls will return the same buffer or the same-sized buffer.

This method never returns <see cref="P:System.Memory`1.Empty" />, but it throw an <see cref="T:System.OutOfMemoryException" /> if the requested buffer size is not available.

You must request a new buffer after calling <see cref="M:System.IO.PipeLines.PipeWriter.Advance(System.Int32)" /> to continue writing more data; you cannot write to a previously acquired buffer.

      ]]></format>
</remarks
<exception cref="T:System.OutOfMemoryException">The requested buffer size is not available.</exception>
```
Can you replace lines 248-252 with the following:
```
<param name="sizeHint">The minimum length of the returned <see cref="T:System.Span`1" />. If 0, a non-empty buffer is returned.</param>
<summary>Returns a <see cref="T:System.Span`1" /> to write to that is at least the requested size, as specified by the <paramref name="sizeHint" /> parameter.</summary>
<returns>A buffer of at least <paramref name="sizeHint" /> bytes. If <paramref name="sizeHint" /> is 0, returns a non-empty buffer.</returns>
<remarks>  
     <format type="text/markdown"><![CDATA[  

There is no guarantee that successive calls will return the same buffer or the same-sized buffer.

This method never returns <see cref="P:System.Span`1.Empty" />, but it throws an <see cref="T:System.OutOfMemoryException" /> if the requested buffer size is not available.

You must request a new buffer after calling <see cref="M:System.IO.PipeLines.PipeWriter.Advance(System.Int32)" /> to continue writing more data; you cannot write to a previously acquired buffer.

      ]]></format>
</remarks>
<exception cref="T:System.OutOfMemoryException">The requested buffer size is not available.</exception>
```

        <summary>To be added.</summary>



         <remarks>To be added.</remarks>
        <remarks>  
          <format type="text/markdown"><![CDATA[  

Text goes here.

         ]]></format>
         </remarks>
```

Reply:
No problem. I'll make the suggested additions in a moment.

Reply:
I have some questions about these suggestions:

1. The markdown code has <see cref="element" /> elements instead of <xref:element>. Are those accepted in markdown too?

2. Where did you get this text from? Did you write it yourself or did it come from somewhere else with existing documentation?

3. The last 3 quoted texts seem to be empty. Did you paste that by mistake or did you intent to paste something different?

Reply:
Thanks for noticing these, @carlossanlop. To answer your questions:

1. These should be xrefs, not crefs. I've changed them.  
2. It came from somewhere else, but I can't find the original source.
3. I pasted it by mistake.

Reply:
No worries, thank you!